FieldOpt/Reservoir/CMakeLists.txt

```diff
@@ -49,4 +54,4 @@ install( TARGETS reservoir
         RUNTIME DESTINATION bin
         LIBRARY DESTINATION lib
         ARCHIVE DESTINATION lib/static
-        )
+        )
```

Comment:
Endline is missing here. It's not a big deal, but there should be a blank line at the end of files. There's a setting in CLion that automatically ensures that there is one.